tests/recovery/recovery_test.go

```diff
@@ -68,8 +68,9 @@ var _ = Describe("Elemental Recovery upgrade tests", func() {
 			s.Reboot()
 			s.EventuallyBootedFrom(sut.Active)
 
-			upgradedVersion := s.GetOSRelease("TIMESTAMP")
-			Expect(upgradedVersion).ToNot(Equal(currentVersion))
+			// TODO check upgrade matches expectations
```

Comment:
With the current change in this PR, the upgrade happens to the same exact image, hence the /etc/os-release will not change after the upgrade. In fact to validate upgrades would be better to test state.yaml file. We could consider adding some helpers to fetch this file in SUT.